OpenRA.Mods.Common/Scripting/Global/MediaGlobal.cs

```diff
@@ -168,24 +169,24 @@ public bool PlayMovieInRadar(string movie, LuaFunction playComplete = null)
 		}
 
 		[Desc("Display a text message to the player.")]
-		public void DisplayMessage(string text, string prefix = "Mission")
+		public void DisplayMessage(string text, string prefix = "Mission", bool useCustomColor = false, HSLColor color = new HSLColor())
```

Comment:
`Microsoft.VisualStudio.Modeling.Diagrams.HslColor.White` is also not an `OpenRA.Graphics.HSLColor` :)

At a more general level, the compiler won't let us do anything like this because non-default parameter values only work for primitive types.